Userland/Libraries/LibGfx/Bitmap.cpp

```diff
@@ -469,6 +469,15 @@ ErrorOr<NonnullRefPtr<Bitmap>> Bitmap::to_bitmap_backed_by_anonymous_buffer() co
     return bitmap;
 }
 
+void Bitmap::invert()
+{
+    for (auto y = 0; y < height(); y++) {
+        for (auto x = 0; x < width(); x++) {
+            set_pixel(x, y, get_pixel(x, y).inverted());
+        }
```

Comment:
no curly braces needed here

Reply:
:laughing: you ninja-ed me, I had this open in a tab & was just about to post in #code_review then this popped up